tests/integration_test/action_service_test.py

```
@@ -14072,6 +14072,136 @@
     assert not DeepDiff(log['llm_logs'][0], expected[0], ignore_order=True, exclude_regex_paths=excludedRegex)
 
 
+@mock.patch.object(litellm, "aembedding", autospec=True)
+def test_prompt_action_response_action_crud_prompt(mock_embedding, aioresponses):
```

Ruff (0.11.9)

Redefinition of unused `aioresponses` from line 7

(F811)